integration-tests/chopsticks/src/network/spiritnet.ts

```diff
@@ -0,0 +1,50 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import { initialBalanceKILT, toNumber } from '../utils.js'
+
+/// Options used to create the Spiritnet context
+const options: SetupOption = {
+	endpoint: process.env.SPIRITNET_WS || 'wss://kilt-rpc.dwellir.com',
+	db: './db/spiritnet.db.sqlite',
+	port: toNumber(process.env.SPIRITNET_PORT) || 9002,
+	wasmOverride: '../../target/debug/wbuild/spiritnet-runtime/spiritnet_runtime.wasm',
+	// Whether to allow WASM unresolved imports when using a WASM to build the parachain. This Flag is needed otherwise, the runtime can not be built from the WASM. Chopsticks throws an error when it encounters an unresolved import.
+	allowUnresolvedImports: true,
+}
+
+/// Assigns the native tokens to an account
+export function assignNativeTokensToAccount(addr: string[], balance: bigint = initialBalanceKILT) {
```

Comment:
```suggestion
export function assignNativeTokensToAccounts(addr: string[], balance: bigint = initialBalanceKILT) {
```

Reply:
updated in: [bec955e](https://github.com/KILTprotocol/kilt-node/pull/614/commits/bec955eb1a2f911de2ef4479826913cacd5e5638)